universal-application-tool-0.0.1/app/models/Account.java

```diff
@@ -58,4 +59,11 @@ public void setManagedByGroup(TrustedIntermediaryGroup group) {
   public Optional<TrustedIntermediaryGroup> getManagedByGroup() {
     return Optional.fromNullable(this.managedByGroup);
   }
+
+  public String getApplicantName() {
```

Comment:
(blocking) An `Account` has many `Applicant`s. Which one is this returning? It looks like it is the most recent applicant? 

Can you add a comment describing which applicant's name this is, and also change the method name to be more specific?

Reply:
ahh that makes sense. i was confused about why we max on time.